pages/about.html

```diff
@@ -4,9 +4,48 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="../styles/style.css" rel="stylesheet"/>
+  <link href="../styles/about.css" rel="stylesheet"/>
+  <title>About</title>
 </head>
 <body>
-  
+  <header>
+    <img src="/assets/in-the-snow.jpg" alt="A woman in the snow" />
+    <h1>Danielle Becker</h1>
+    <nav>
+      <ul>
+        <li><a class="header" href="index.html">Home</a></li>
+        <li><a class="header" href="about.html">About</a></li>
+        <li><a class="header" href="portfolio.html">Portfolio</a></li>                   
```

Comment:
When you move the index.html file to the root of the projects the paths of the files would need to be updated.